src/gui/runtime/plugin/nvim_gui_shim.vim

```diff
@@ -206,3 +206,13 @@ endfunction
 command! GuiTreeviewToggle call <SID>TreeViewToggle()
 noremap <script> <Plug>GuiTreeviewToggle :call <SID>TreeViewToggle()
 anoremenu <script> Gui.Treeview.Toggle :call <SID>TreeViewShowToggle()
+
+" Show Right-Click ContextMenu
+function s:GuiShowContextMenu() range
+	call rpcnotify(0, 'Gui', 'ShowContextMenu')
+endfunction
+
+command! -range GuiShowContextMenu call s:GuiShowContextMenu()
+nnoremap <silent><RightMouse> :GuiShowContextMenu<CR>
```

Comment:
Can we keep these disabled by default, for now it is enough to move these examples to the docs/README.

... which reminds me, the docs :D https://github.com/equalsraf/neovim-qt/blob/master/src/gui/runtime/doc/nvim_gui_shim.txt

Reply:
Sure! That makes sense to me :)

I can get rid of the `command! GuiShowContextMenu`, and add a brief summary to the documentation.

I don't see a `docs/README`, so I assume the example should be added to `nvim_gui_shim.txt`?

```
							*GuiShowContextMenu()*
Displays a Cut-Copy-Paste context menu at the current cursor position. This
event can be bound to right click events in ginit.vim, e.g.
>
	nnoremap <silent><RightMouse> :call GuiShowContextMenu()<CR>
	inoremap <silent><RightMouse> <Esc>:call GuiShowContextMenu()<CR>
	vnoremap <silent><RightMouse> :call GuiShowContextMenu()<CR>gv
```